roles/monitoring/defaults/main.yaml

```diff
@@ -8,19 +8,20 @@ V4M_NODE_PLACEMENT_ENABLE: false
 V4M_BASE_DOMAIN: "{{ V4_CFG_BASE_DOMAIN }}"
 V4M_CERT: null
 V4M_KEY: null
+V4M_KB_KNOWN_NODEPORT_ENABLE: false
 
 V4M_LOGGING_NAMESPACE: logging
 V4M_MONITORING_NAMESPACE: monitoring
 
-V4M_KIBANA_FQDN: "kibana.{{ V4M_BASE_DOMAIN }}"
+V4M_KIBANA_FQDN: "dashboards.{{ V4M_BASE_DOMAIN }}"
 V4M_KIBANA_CERT: "{{ V4M_CERT }}"
 V4M_KIBANA_KEY: "{{ V4M_KEY }}"
 V4M_KIBANA_PASSWORD: "{{ lookup('password', '/dev/null chars=ascii_letters,digits') }}"
 V4M_KIBANASERVER_PASSWORD: "{{ lookup('password', '/dev/null chars=ascii_letters,digits') }}"
 V4M_LOGCOLLECTOR_PASSWORD: "{{ lookup('password', '/dev/null chars=ascii_letters,digits') }}"
 V4M_METRICGETTER_PASSWORD: "{{ lookup('password', '/dev/null chars=ascii_letters,digits') }}"
 
-V4M_ELASTICSEARCH_FQDN: "elasticsearch.{{ V4M_BASE_DOMAIN }}"
+V4M_ELASTICSEARCH_FQDN: "search.{{ V4M_BASE_DOMAIN }}"
```

Comment:
Good catch, I'll update those two default values in CONFIG-VARS.md to indicate that change.
Regarding the general question, for now, the V4M team has suggested leaving all the V4M variable names as they are. They are planning an update in the short term so that they will better match the underlying log/mon software that is being installed.

Reply:
CONFIG-VARS.md "default" column has been updated for V4M_KIBANA_FQDN & V4M_ELASTICSEARCH_FQDN